README.md

```diff
@@ -103,3 +73,10 @@ knative-serving-operator-8c7858985-v7zrd       1/1       Running   0          2m
 openshift-pipelines-operator-c56876c69-6jwqw   1/1       Running   0          2m
 tekton-pipelines-controller-5576fbb979-mv8hp   1/1       Running   0          55s
 tekton-pipelines-webhook-78bf9c5f46-9pzlj      1/1       Running   0          55s
+```
```

Comment:
The example oc command outputs above seem to be a bit outdated also.

Reply:
OK I put new output in.  I left out the tekton dashboard and webhook, since someone doing this manually doesn't have the instructions to install these (the `dependencies.yaml` doesn't include the dashboard or webhook extension).